packages/core/src/runtime/runtimeAstTransformer.ts

```diff
@@ -112,6 +114,19 @@ export class RuntimeAstTransformer extends ASTVisitor<undefined, ExecutableExpre
         );
     }
 
+    override visitOperatorExpression(expression: OperatorExpression): ExecutableExpression<any> {
+        return new ExecutableOperatorExpression(
+            this.optionalExpression(expression),
+            this.visit(expression.left),
+            this.visit(expression.right),
+            this.visit(expression.operator)
+        );
+    }
+
+    override visitNoopExpression(): ExecutableExpression<any> {
+        throw new Error("NoopExpression cannot be transformed to executable expression.");
+    }
```

Comment:
it's just a placeholder due to fault tolerant parsing
previously, operator calls were transformed to simple call expressions
in fault-tolerant mode, this would then just have resulted in a call expression with one argument instead of 2
however, now the "real" operator expression require exactly two "arguments"
I thought about making the second one optional for fault tolerant mode, but as executing this without the second one would always be incorrect, I decided against it
even in fault tolerant mode, the execution stops before (and should never execute the function call caused by the operator)
I see two good options
1. rename the noop expression (suggestions welcome)
2. make it a noop. However, then the error of executing it would not be cought

Reply:
> 3. Return an Optional containing an expression to execute

is not an option?

Reply:
Other than that, I'm also fine with the current approach of disallowing noops at runtime

Reply:
> > 3. Return an Optional containing an expression to execute
> 
> is not an option?

js/ts has no optionals by default
we could create our own, however this would be more or less equivalent to the previous make it nullable approach
I don't really like this one, because the contract of an executable operator expression is two have two operands, not 1 or 2
When this would be encountered at runtime, what to do? Throw an error then? Or fallback to `null`? In case of an error, I prefer the noop expression as this once could be reused for similar cases in the future